autotest/osr/osr_basic.py

```diff
@@ -1663,3 +1663,13 @@ def test_osr_GetUTMZone_Projected3D():
     utm_srs.PromoteTo3D()
 
     assert utm_srs.GetUTMZone() == -11
+
+
+###############################################################################
+def test_SetPROJAuxDbPaths():
+
+    osr.SetPROJAuxDbPath('../cpp/data/test_aux.db')
```

Comment:
I would avoid doing that in the main process as it will indirectly influence all later tests. For a similar situation, we have gcore/basic_test_subprocess.py that is called by gcore/basic_test.py . And perhaps just calling SetPROJAuxDbPath with a dummy path and see that something afterwards doesn't crash would be enough